plugins/main/public/components/common/wazuh-discover/wz-flyout-discover.tsx

```diff
@@ -307,21 +306,20 @@ const WazuhFlyoutDiscoverComponent = (props: WazuhDiscoverProps) => {
                         : undefined
                     }
                   />
-                  {absoluteDateRange ? (
-                    <EuiFlexGroup
-                      gutterSize='s'
-                      responsive={false}
-                      justifyContent='center'
-                      alignItems='center'
-                    >
-                      <EuiFlexItem grow={false}>
-                        <EuiText size='s'>
-                          {formatUIDate(absoluteDateRange?.from)} -{' '}
-                          {formatUIDate(absoluteDateRange?.to)}
-                        </EuiText>
-                      </EuiFlexItem>
-                    </EuiFlexGroup>
-                  ) : null}
+
+                  <EuiFlexGroup
+                    gutterSize='s'
+                    responsive={false}
+                    justifyContent='center'
+                    alignItems='center'
+                  >
+                    <EuiFlexItem grow={false}>
+                      <EuiText size='s'>
+                        {formatUIDate(dateRange?.from)} -{' '}
```

Comment:
**nitpick**: using a relative date such as `Last 24 hours`, this display `Invalid date - Invalid date`:
![image](https://github.com/user-attachments/assets/642993ca-1d47-408b-95db-b2f2b4c6f2f8)